app/src/main/java/com/example/weather/network/CachedCityDetails.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.weather.network
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "city_table")
+data class CachedCityDetails(
+    @PrimaryKey val name: String,
+    val country: String,
+    val temp: Double,
+    val temp_max: Double,
+    val temp_min: Double,
+    val humidity: Int,
+    val wind: Double,
+    val pressure: Int,
+    val sunrise: Int,
+    val sunset: Int,
+    val icon: String,
+    val description: String
+)
+{
+    fun toDomainModel(): CityDetails {
+        return CityDetails(
+            base="",
```

Comment:
Let's make this empty string a constant and move it to a `constants.kt` file since it'll be used throughout the app
same with the -1's below